packages/core/src/Badge/Badge.styles.tsx

```diff
@@ -25,7 +25,7 @@ export const { staticClasses, useClasses } = createClasses("HvBadge", {
     ...theme.typography.caption2,
     color: theme.colors.textDimmed,
     borderRadius: theme.radii.full,
-    backgroundColor: theme.colors.text,
+    backgroundColor: "var(--bg-color)",
```

Comment:
Can we optimise this to fallback to `text`? This way we can remove the default value, and we get less code all around (no default prop, no CSS var, etc.)
```suggestion
    backgroundColor: `var(--bg-color, ${theme.colors.text})`,
```